internal/txlib/credentials.go

```diff
@@ -1,7 +1,9 @@
 package txlib
 
 import (
+	"errors"
 	"fmt"
+
```

Comment:
I am not sure if this break line is needed

Reply:
[gofmt](https://pkg.go.dev/cmd/gofmt) adds it automatically. I guess it is supposed to separate built-in libraries from user-created libraries.